code/lightna/magento-os-frontend/webpack.yaml

```diff
@@ -10,3 +10,5 @@ entry:
       ProductOptions: product/Options
       Gallery: product/Gallery
       Facets: listing/Facets
+    module:
+      updateMagentoCartSectionId: common/updateMagentoCartSectionId
```

Comment:
it seems we don't need it anymore, it was needed when we have both magento and lightna minicart on the website but now we have globally our header

Reply:
hmm... but what if we don't use semi-lightna? maybe we should remove it from common entry in semi-module? it is possible?

Reply:
it's hard to imagine such case on real project, this case means that Lightna pages and Magento pages have different headers

Reply:
let's remove it completely than

Reply:
I'll move into separate PR if you think too long on another thread 😃